BrowserServicesKit/Sources/NetworkProtection/ExtensionMessage/ExtensionRequest.swift

```diff
@@ -24,7 +24,7 @@ public enum VPNCommand: Codable {
     case removeVPNConfiguration
     case sendTestNotification
     case restartAdapter
-    case uninstallVPN
+    case uninstallVPN(showNotification: Bool)
```

Comment:
Had to add support for NOT showing the update notification, since when updating through the new message it was a bit confusing to users - as they didn't think they were uninstalling the VPN, which is correct.

Reply:
No - it is an intentional choice not to show the uninstall message to users.